dts/bindings/mtd/nxp,s32-qspi-hyperflash.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  QSPI hyperflash connected to the NXP S32 QSPI bus.
+
+compatible: "nxp,s32-qspi-hyperflash"
+
+include: ["nxp,s32-qspi-device.yaml", "soc-nv-flash.yaml"]
+
+properties:
+  program-page-size:
+    type: int
+    default: 256
+    description: The programing page size in byte
+
+  device-id:
+    type: uint8-array
+    required: true
+    description: Device memory ID
+
+  device-id-word-addr:
+    required: true
+    type: int
+    description: The word address of the device ID in ASO
+
+  rwds-low-dual-error:
+    type: boolean
+    description: Enabled RWDS dual error detect
+
+  secure-region-locked:
+    type: boolean
+    description: Locked secure region
+
+  drv-strength:
+    type: string
+    enum:
+      - DRV STRENGTH 000 # Typical impedance for 1.8V: 27, typical impedance 3V: 20
```

Comment:
move the comments to the description, so that they appear in the binding docs

Reply:
I updated